ignore/src/types.rs

```diff
@@ -196,6 +196,7 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
         "OFL-*[0-9]*",
     ]),
     ("lisp", &["*.el", "*.jl", "*.lisp", "*.lsp", "*.sc", "*.scm"]),
+    ("lockfiles", &["composer.lock", "package-lock.json", "yarn.lock", "Gemfile.lock"]),
```

Comment:
Maybe call this `lock` instead? It's a bit shorter.

Also, you might as well add `Cargo.lock` to this.

And please make sure the style stays consistent with the surrounding code. i.e., No lines over 79 columns please.